### PR TITLE
Add "Unspecified" Build Type and Use by Default (Kits Only)

### PR DESCRIPTION
**Pull request by snehara99**
For #3821 

This makes it so that the default variant is "Unspecified" which basically lets CMake handle what build type to use.

### CODE REVIEW

File: src/rollbar.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ class RollbarController {
      */
     error(what: string, additional: object = {}): void {
         log.error(what, JSON.stringify(additional));
-        debugger;
+        // debugger;
```
Comment (gcampbell-msft) [inline src/rollbar.ts:87]: Why this change?

Comment (snehara99) [reply src/rollbar.ts:87]: oh whoops sorry I was supposed to revert that. It was for testing. Will fix

Comment (snehara99) [reply src/rollbar.ts]: shoudl be fixed

**Comment by snehara99** [comment]
@sinemakinci1 this had a docs change if you want to review

**Comment by snehara99** [comment]
@gcampbell-msft What do you think of the wording updates in the doc file update? I made a distinction between the word "option" and "variant" that may be confusing. Do you have ideas for alternative wording?

**Review by gcampbell-msft — DISMISSED** [review]

**Review by gcampbell-msft — APPROVED** [review]
